src/functions.php

```diff
@@ -736,3 +736,30 @@ function parse($value)
 {
     return Context::get()->getConfig()->parse($value);
 }
+
+function locateBinaryPath($name)
+{
+    $nameEscaped = escapeshellarg($name);
+
+    // Try `command`, should cover all Bourne-like shells
+    if (commandExist("command")) {
+        return run("command -v $nameEscaped")->toString();
+    }
+
+    // Try `which`, should cover most other cases
+    if (commandExist("which")) {
```

Comment:
Again, sorry, but how do we use docker for what? Deployments? We have couple of projects using deployer and as well dockerized. CI is then using Deployer to deploy to environments at various stages of the project from within docker, using a build file in each project. Deployer then is run from within docker, as is pretty much every other dependency of the project, including phpunit, etc.